libs/core/src/components/pds-tabs/pds-tab/pds-tab.scss

```diff
@@ -1,4 +1,24 @@
 pds-tab {
+  --background-color-tab: var(--pine-color-neutral-grey-300);
+  --background-color-tab-active: var(--pine-color-neutral-charcoal-400);
+  --background-color-tab-hover: var(--pine-color-neutral-grey-400);
+  --border-radius: var(--pine-border-radius-xs);
+
+  --color: var(--pine-color-neutral-charcoal-200);
+  --color-active: var(--pine-color-neutral-charcoal-400);
+  --color-focus: var(--pine-color-neutral-charcoal-300);
+  --color-white: var(--pine-color-base-white);
+
+  --font-size-body: var(--pine-font-size-body);
+  --font-size-sm: var(--pine-font-size-body-sm);
+  --font-size-xs: var(--pine-font-size-body-xs);
+  --font-weight: var(--pine-font-weight-medium);
+  --line-height-md: var(--pine-line-height-md);
+  --line-height-sm: var(--pine-line-height-sm);
+  --line-height-xs: var(--pine-line-height-xs);
+
+  --outline: 4px solid var(--pine-color-primary-200);
```

Comment:
Is there a reason you didn't use `var(--outline)` here like you did below?

Reply:
Not sure I understand the question here. Can you clarify?

Reply:
NVM... 🤦🏾 It was early in the morning when I looked at this again. Ignore. This was the definition for the variable.